File: demo-vk-rest-scraping.java/src/main/java/demo/model/Item.java
package demo.model;

import com.google.api.client.util.Key;

public class Item {

    @Key("cid")
    private int cid;

    @Key("title")
    private String title;

    public int getCid() {
        return cid;
    }

    public void setCid(int cid) {
        this.cid = cid;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    @Override
    public boolean equals(Object obj) {
        if (this == obj)
            return true;
        if (obj == null || getClass() != obj.getClass())
            return false;

        Item item = (Item) obj;

        if (cid != item.cid)
            return false;
        return title.equals(item.title);
    }

    @Override
    public int hashCode() {
        int result = cid;
        result = 31 * result + title.hashCode();
        return result;
    }

    @Override
    public String toString() {
        return "Country{" +
                "cid='" + cid + '\'' +
                ", title='" + title + '\'' +
                '}';
    }
}

File: demo-vk-rest-scraping.kotlin/src/main/kotlin/demo/model/Item.kt
package demo.model

import com.google.api.client.util.Key

class Item {

    @Key("cid")
    var cid: Int = 0

    @Key("title")
    var title: String? = null

    override fun equals(other: Any?): Boolean {
        if (this === other)
            return true
        if (other == null || javaClass != other.javaClass)
            return false

        val item = other as Item?

        if (cid != item!!.cid)
            return false
        return title == item.title
    }

    override fun hashCode(): Int {
        var result = cid
        result = 31 * result + title!!.hashCode()
        return result
    }

    override fun toString(): String {
        return "Country{" +
                "cid='" + cid + '\'' +
                ", title='" + title + '\'' +
                '}'
    }
}
